src/test/java/com/WWI16AMA/backend_api/PlaneTests.java

```diff
@@ -174,7 +174,7 @@ public void testPostPlaneLog() throws Exception {
         FlightAuthorization.Authorization auth = FlightAuthorization.Authorization.PPLB;
         PlaneLogEntry planeLogEntry = new PlaneLogEntry(LocalDateTime.of(2019, 3, 12, 14, 55, 13), 0, "TestOrt", 69, 88, 5);
 
-        this.mockMvc.perform(put("/planeLog/1" + TestUtil.getUnusedId(planeRepository))
+        this.mockMvc.perform(put("/planeLog/" + TestUtil.getUnusedId(planeRepository))
```

Comment:
Der Test hier schlägt fehl, weil er als Fehlerstatus eine 405 statt einer 404 bekommt.
`405 Method Not Allowed` kommt hier zustande, weil du `put("/planeLog/{ungültigeId}")` statt `post("[...]")` aufrufst.

Es gibt bei `mockMvc.perform()` tatsächlich `get(), put(), post() und delete()`
Die relevanten API-Docs sind zum [MockMvc](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/MockMvc.html) und zum [MockMvcRequestBuilders](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/request/MockMvcRequestBuilders.html).

Also einfach `put` -> `post` und alles läuft supi..

Reply:
Ganz cool wären außerdem noch ein Test für den positiv-Fall und dann ein Test für deine Sanity-Checks. Meine Empfehlung: Erst die Tests und dann das Feature entwickeln.